test/Route.test.js

```diff
@@ -103,13 +105,17 @@ describe('Route', () => {
       });
 
       it('should set operation variables', () => {
-        const operationVariables = [
-          {
-            name: 'id',
+        const name = 'id';
```

Comment:
Looks like this is no longer an array but instead an object with [name]: VariableInfo